mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/KitPvpManager.java

```diff
@@ -112,21 +143,32 @@ public Kit createKit(Transaction transaction, String name, Set<ItemInSlot> conte
 		}
 
 		Kit kit = new Kit(kitId, name, contents);
-		existingKits.put(kitId, kit);
+		existingKits.put(new KitKeyName(name), kit);
```

Comment:
There should be 2 cache entries added here:
```suggestion
		existingKits.put(new KitKeyName(name), kit);
		existingKits.put(new KitKeyId(kitId), kit);
```